include/flatbuffers/flatbuffers.h

```diff
@@ -1470,6 +1502,21 @@ class FlatBufferBuilder
     finished = true;
   }
 
+  // Allocates space for a vector of structures.
+  // Must be completed with EndVectorOfStructs().
+  template<typename T>
```

Comment:
Sure, done.  Though I notice that we have a load of other stuff under protected that probably shouldn't be there.